projects/packages/boost-core/src/lib/class-utils.php

```diff
@@ -38,7 +38,7 @@ public static function standardize_error( $error ) {
 		if ( is_object( $error ) ) {
 			return array(
 				'name'    => 'Error',
-				'message' => json_decode( wp_json_encode( $error ), true ),
+				'message' => json_decode( wp_json_encode( $error, JSON_UNESCAPED_SLASHES ), true ),
```

Comment:
Since this is just being decoded right away we could include `JSON_UNESCAPED_UNICODE` here. Seems like it might save a few nanoseconds.
```suggestion
				'message' => json_decode( wp_json_encode( $error, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE ), true ),
```

Reply:
24d018fdee